### PR TITLE
Was added format applying for hints

### PR DESCRIPTION
**Pull request by EugeneElkin**

### CODE REVIEW

File: src/visual.ts [hunk 1]
```diff
@@ -680,6 +680,8 @@ module powerbi.extensibility.visual {
                 return;
             }
 
+            let dataViewValues: any = visualUpdateOptions.dataViews[0].categorical.values[0];
```
Comment (ignatvilesov) [inline src/visual.ts]: It'd be better to allow users to control display units and precision via formatting panel.

File: src/visual.ts [hunk 2]
```diff
@@ -1503,10 +1505,20 @@ module powerbi.extensibility.visual {
                 .each("end", callback);
         }
         private renderTooltip(selection: UpdateSelection<WordCloudDataPoint>): void {
+            let categorical: WordCloudColumns<DataViewCategoryColumn> = WordCloudColumns.getCategoricalColumns(this.incomingUpdateOptions.dataViews[0]),
+            wordValueFormatter: IValueFormatter = null;
+
+            if (categorical.Values && categorical.Values[0]) {
+                wordValueFormatter = ValueFormatter.create({
+                    format: ValueFormatter.getFormatStringByColumn(categorical.Values[0].source)
```
Comment (ignatvilesov) [inline src/visual.ts]: I think it'd be better to specify [culture selector](https://github.com/Microsoft/powerbi-visuals-utils-formattingutils/blob/master/lib/index.d.ts#L749) as well

File: src/visual.ts [hunk 3]
```diff
@@ -1503,10 +1505,20 @@ module powerbi.extensibility.visual {
...
+                });
+            }
+
             this.tooltipService.addTooltip(selection, (tooltipEvent: TooltipEventArgs<WordCloudDataPoint>) => {
+                let item = wordValueFormatter !== null ? wordValueFormatter.format(tooltipEvent.data.count) : tooltipEvent.data.count;
                 return [{
                     displayName: tooltipEvent.data.text,
-                    value: tooltipEvent.data.count.toString()
+                    value: item.toString()
```
Comment (ignatvilesov) [inline src/visual.ts]: Please increase version of the visual if you are going to publish it.

File: src/visual.ts (hunk 1 above)
Comment (EugeneElkin) [reply src/visual.ts]: It's unnecessary line. I forgot to remove it.

File: src/visual.ts (hunk 3 above)
Comment (EugeneElkin) [reply src/visual.ts]: Oh, sure. Sorry.

File: src/visual.ts (hunk 1 above)
Comment (ignatvilesov) [reply src/visual.ts]: Sorry. There's no need to implement this for tooltips.

File: src/visual.ts (hunk 2 above)
Comment (EugeneElkin) [reply src/visual.ts]: Is there examples how to use it? I don't see that another formatters in visual use it.

**Pull request merged**